메인화면 구성

HyunBaeJung opened the pull request:
main page
[] ideas

chatroom side bar
[ v ] layout
[] copy discord hover effect
[] D&D action
[] random image create

HyunBaeJung committed:
메인화면 구성

main page
[] ideas

chatroom side bar
[ v ] layout
[] copy discord hover effect
[] D&D action
[] random image create

HyunBaeJung committed:
fe in progress

sinna94 reviewed (approved):
👍 구성 끝나면 캡처 한장 넣어주십셔

HyunBaeJung committed:
fe dev

sidebar animation complete

HyunBaeJung committed:
메인페이지 개발중

HyunBaeJung committed:
메인페이지 개발

■ 채팅방 디스플레이 레이아웃
■ 랜덤생성 이미지 개발

HyunBaeJung committed:
로그인/회원가입 화면 개선

디자인 개선
form적용, enter로 submit 추가.

HyunBaeJung committed:
사이드바 + 채팅방생성

HyunBaeJung committed:
dev ing

HyunBaeJung committed:
로그인시 렌더링 시점 해결

useEffect 두번째 인수를 빈 배열이 아니라 빈 배열에 로그인 여부 state를 담아서 선언하여

■기존
- 화면 렌더링 때마다 조회

■기존
- 로그인 성공 시 조회
- 화면 렌더링 때마다 조회

으로 변경
